--- a/deployment/common/os_detect.sh
+++ b/deployment/common/os_detect.sh
@@ -24,6 +24,8 @@ function detect ()
         OS="Ubuntu 14.04"
     elif [ `echo "${input}" | grep "Red Hat" | grep "6." | wc -l` = "1" ]; then
         OS="RedHat 6"
+    elif [ `echo "${input}" | grep "Debian" | grep "8." | wc -l` = "1" ]; then
+        OS="Ubuntu 14.04"       # Report Ubuntu 14.04 for Debian 8
     fi
 }
 
