--- a/debian/control.trusty.in
+++ b/debian/control.trusty.in
@@ -19,8 +19,8 @@ Homepage: http://scidb.org
 Package: PKGNAME-VERSION_MAJOR.VERSION_MINOR
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, python-paramiko, python-crypto
-Recommends: postgresql-client-9.3
-Suggests: postgresql-9.3, postgresql-contrib-9.3
+Recommends: postgresql-client-9.4
+Suggests: postgresql-9.4, postgresql-contrib-9.4
 Description: SciDB - database for extra large data processing
     BUILD
 
@@ -110,6 +110,6 @@ Description: Install entire SciDB
 
 Package: PKGNAME-VERSION_MAJOR.VERSION_MINOR-all-coord
 Architecture: any
-Depends: PKGNAME-VERSION_MAJOR.VERSION_MINOR (= ${binary:Version}), PKGNAME-VERSION_MAJOR.VERSION_MINOR-client (= ${binary:Version}), PKGNAME-VERSION_MAJOR.VERSION_MINOR-utils (= ${binary:Version}), PKGNAME-VERSION_MAJOR.VERSION_MINOR-plugins (= ${binary:Version}), PKGNAME-VERSION_MAJOR.VERSION_MINOR-client-python (= ${binary:Version}), postgresql-client-9.3, postgresql-9.3, postgresql-contrib-9.3, sudo
+Depends: PKGNAME-VERSION_MAJOR.VERSION_MINOR (= ${binary:Version}), PKGNAME-VERSION_MAJOR.VERSION_MINOR-client (= ${binary:Version}), PKGNAME-VERSION_MAJOR.VERSION_MINOR-utils (= ${binary:Version}), PKGNAME-VERSION_MAJOR.VERSION_MINOR-plugins (= ${binary:Version}), PKGNAME-VERSION_MAJOR.VERSION_MINOR-client-python (= ${binary:Version}), postgresql-client-9.4, postgresql-9.4, postgresql-contrib-9.4, sudo
 Description: Install entire SciDB and PostgreSQL
     BUILD
