--- a/run.py
+++ b/run.py
@@ -661,7 +661,7 @@ def install(scidbEnv):
     os.chdir(scidbEnv.build_path)
 
     if not scidbEnv.args.light:
-        rm_rf(scidbEnv.install_path+"/*", scidbEnv.args.force)
+        rm_rf(scidbEnv.install_path+"/*", scidbEnv.args.force, throw=False)
 
     mkdir_p(scidbEnv.install_path)
 
@@ -681,7 +681,7 @@ def install(scidbEnv):
         ret = executeIt(cmdList)
         data_path = getDataPath(configFile, db_name)
     else:
-        data_path=os.path.join(scidbEnv.stage_path, "DB-"+db_name)
+        data_path=os.path.join(scidbEnv.install_path, "DB-"+db_name)
         data_path=os.environ.get("SCIDB_DATA_PATH", data_path)
         instance_num=int(os.environ.get("SCIDB_INSTANCE_NUM","4"))
         port=int(os.environ.get("SCIDB_PORT","1239"))
